source/atlas-cli-deploy-local.txt

```diff
@@ -409,6 +409,16 @@ atlas deployments respectively.
       #. Specify how you want to connect to the deployment and press 
          :kbd:`Enter`..
 
+         You can connect with any of the following or retrieve the
```

Comment:
nit - clarify that the following list refers to clients not connection string retrieval methods
```suggestion
         You can retrieve the connection string or connect to the following clients:
```